plugins/scaffolder-react/src/next/components/TemplateCard/TemplateCard.tsx

```diff
@@ -174,6 +174,7 @@ export const TemplateCard = (props: TemplateCardProps) => {
                   style={{ marginLeft: '8px' }}
                   entityRefs={ownedByRelations}
                   defaultKind="Group"
+                  hideIcons
```

Comment:
tempting to remove the UserIcon instead eh

Reply:
Yeah, was gonna leave this for now, just because it was a bit awkward to restyle and such. Can revisit though later.